plugin/trino-pinot/src/test/java/io/trino/plugin/pinot/TestPinotIntegrationSmokeTest.java

```diff
@@ -796,4 +860,264 @@ public void testLimitPushdown()
         assertThat(query("SELECT string_col, long_col FROM " + ALL_TYPES_TABLE + "  WHERE int_col >0 AND bool_col = 'false' LIMIT " + MAX_ROWS_PER_SPLIT_FOR_SEGMENT_QUERIES))
                 .isNotFullyPushedDown(LimitNode.class);
     }
+
+    @Test
+    public void testAggregationPushdown()
```

Comment:
Take a look at `BaseJdbcConnectorTest#testAggregationPushdown` (and similarly named methods) for some other useful tests.

Reply:
Sounds good! I will use the base connector tests as soon as pinot supports varbinary and boolean types, which will be in the next release.

Reply:
Sounds good! I will use the base connector tests as soon as pinot supports varbinary and boolean types, which will be in the next release.